### PR TITLE
Housing Changes

### PR DESCRIPTION
**Pull request by weber530**
Gets all room data from the database and displays under 'Housing Group Management'. Also, it displays floor plans for each floor and displays appropriate rooms for the floor. In the future, I'll work on keeping my commits/pushes more frequent and more focused on one thing.

### CODE REVIEW

File: site/RoomDraw/src/app/housing/housing.component.ts [hunk 1]
```diff
@@ -1,27 +1,94 @@
...
 	selector: 'app-housing',
 	templateUrl: './housing.component.html',
-	styleUrls: ['./housing.component.css']
+	styleUrls: ['./housing.component.scss']
 })
 export class HousingComponent implements OnInit {
+	rooms: Room[] = []
+	counter = 0
 
 	halls = [
```
Comment (ForTheReallys) [inline site/RoomDraw/src/app/housing/housing.component.ts:14]: This data should be pulled from the api.

**Comment by ForTheReallys** [comment]
This looks good.  I requested a couple of changes above.  Let me know when you start.  I'll make two more changes to this.  If I finish those two changes before you see this, I'll make the requested change.

File: site/RoomDraw/src/app/housing/housing.component.ts (hunk 1 above)
Comment (ForTheReallys) [reply site/RoomDraw/src/app/housing/housing.component.ts:14]: To be clear, the `halls` variable shoudl be pulled from the api

**Comment by weber530** [comment]
Okay, I'm getting started on this.

**Comment by ForTheReallys** [comment]
No worries. I already merged

**Pull request merged**